llumnix/arg_utils.py

```diff
@@ -212,5 +228,8 @@ def add_cli_args(
                             type=int,
                             default=EngineManagerArgs.max_stages,
                             help='drop migration if the number of stages > max_stages')
-
+        parser.add_argument("--pdd-config",
```

Comment:
I'd prefer a general config file than a special pdd config. We'd have more configs in the future. Just make pdd config an entry of a general config json.